CHANGES.md

```diff
@@ -7,7 +7,7 @@ Release Notes.
 #### Features
 * Support `NETWORK` Profiling.
 * Let the logger as a configurable module.
-* Support analyze the data of OpenSSL library in `NETWORK` Profiling.
+* Support analyze the data of OpenSSL, BoringSSL library in `NETWORK` Profiling.
```

Comment:
Do we have a doc about network profiling in the rover or backend? If so, we should update the doc about this too.

Reply:
No, we haven't the doc to explain this. Update the document to explain which type of profiling the rover is supported?

Reply:
Explain which protocols/deployments we could support. I think we have ON and OFF CPU profiling in rover, right?

Reply:
I have updated the configuration file to explain the ON/OFF CPU and Network profiling task.